ios/FluentUI/Vnext/List/ListTokens.swift

```diff
@@ -39,40 +39,67 @@ import SwiftUI
 // MARK: ListCell Tokens
 
 open class CellBaseTokens: ControlTokens {
+    /// Defines the size of the leading view of the Cell.
     public internal(set) var cellLeadingViewSize: MSFListCellLeadingViewSize = .medium
 
+    /// Defines the foreground color of the disclosure icon of the Cell.
     open var disclosureIconForegroundColor: DynamicColor { aliasTokens.foregroundColors[.neutral4] }
 
+    /// Defines the color of the label of the Cell.
     open var labelColor: DynamicColor { aliasTokens.foregroundColors[.neutral1] }
 
+    /// Defines the color of the label when the Cell is selected.
+    open var labelSelectedColor: DynamicColor { aliasTokens.foregroundColors[.brandRest] }
+
+    /// Defines the color of the leading view of the Cell.
     open var leadingViewColor: DynamicColor { aliasTokens.foregroundColors[.neutral1] }
 
+    /// Defines the color of the sublabel of the Cell.
     open var sublabelColor: DynamicColor { aliasTokens.foregroundColors[.neutral3] }
 
+    /// Defines the color of the sublabel when the Cell is selected.
+    open var sublabelSelectedColor: DynamicColor { aliasTokens.foregroundColors[.brandRest] }
```

Comment:
Oh interesting. Again for my information mostly, but why are these defined as DynamicColors in the Tokens class, when they're used as just Colors in code?

Reply:
I think my answer to your other question should cover this, but all of the tokens defined as `DynamicColor` need to eventually end up as either `Color` or `UIColor`. I guess a lot of the `UIColor` needs to then also change to `CGColor`, too. If you look at the usage, you should see a bunch of `Color(dynamicColor: )` or `UIColor(dynamicColor: )`

Reply:
Most of the color classes we deal with in iOS are either toolkit-specific (`UIColor` in UIKit on iOS, `NSColor` in AppKit on macOS, `Color` in SwiftUI on both), or too simple (`CGColor` has no concept of multiple color values for different UI states). So as part of the new token work we created a new platform-agnostic struct called `DynamicColor` that can act as a superset of all needed functionality for all platforms/toolkits. Then we have specific converters that convert `DynamicColor` values to the color type expected in our toolkit of choice.